Gemfile

```diff
@@ -10,8 +10,8 @@ group :test do
   gem 'hashdiff'
   gem 'minitest'
   gem 'mirage'
-  gem 'rake'
-  gem 'pronto'
+  gem "rake"
+  gem "pronto"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pronto` should appear before `rake`.